modules/openxr/openxr_api.h

```diff
@@ -46,17 +46,43 @@
 #include "core/templates/rb_map.h"
 #include "core/templates/rid_owner.h"
 #include "core/templates/vector.h"
+#include "servers/rendering_server.h"
 #include "servers/xr/xr_pose.h"
 
 #include <openxr/openxr.h>
 
-// Note, OpenXR code that we wrote for our plugin makes use of C++20 notation for initializing structs which ensures zeroing out unspecified members.
-// Godot is currently restricted to C++17 which doesn't allow this notation. Make sure critical fields are set.
+// TODO should move this into rendering_server.h ?
+#ifdef DEBUG_ENABLED
+#define ERR_ON_RENDER_THREAD                                              \
```

Comment:
This should probably move into `rendering_server.h` but I noticed that a lot of the other macros (that only work internally) are in a separate include.